Resources/Prototypes/Voice/speech_emotes.yml

```diff
@@ -176,3 +176,17 @@
   chatMessages: [ whirrs ]
   chatTriggers:
     - whirrs
+
+- type: emote
+  id: Mars
+  category: Vocal
+  chatMessages: [ mars ]
+  chatTriggers:
+    - mars
+
+- type: emote
+  id: Wurble
+  category: Vocal
+  chatMessages: [ wurble ]
+  chatTriggers:
+    - wurble
```

Comment:
Add these to the Harpy emote list! Remember the Birbs should be able to imitate the funny Shadowkin.
![Farida Sing 250 transparent](https://github.com/user-attachments/assets/3aab6b99-74e4-4033-883a-fff22972b71d)

Reply:
Those were already added to Harpies if i recall.